Shared/Common/Helpers/AppearanceHelper.swift

```diff
@@ -9,19 +9,12 @@ class AppearanceHelper {
     static let shared = AppearanceHelper()
 
     func setupAppearance() {
-        let navBarImage = UIImage.createGradientImage(
-            frame: CGRect(x: 0, y: 0, width: UIScreen.main.bounds.width, height: UIScreen.main.bounds.height),
-            colors: [Constant.color.lockBoxBlue, Constant.color.lockBoxTeal],
-            locations: [0.15, 0]
-        )
-
-        UINavigationBar.appearance().barTintColor = UIColor(patternImage: navBarImage!)
-        UINavigationBar.appearance().isTranslucent = true
+        UINavigationBar.appearance().barTintColor = Constant.color.navBackgroundColor
+        UINavigationBar.appearance().isTranslucent = false
         UINavigationBar.appearance().prefersLargeTitles = true
```

Comment:
To @nickbrandt‘s point about large title on the home screen, and this may already be super obvious to you @joeyg, but when I implemented the “large title” feature on the detail view at #259 I remember that I (think I) had to have the default be “true” here and then go “unset it” (`.never`) it everywhere else...

Reply:
@joeyg ah so the way I see it on my device, is it because this large title style is required by the new search and how we do pull to refresh on the table?

In other words, setting `largeTitleDisplayMode` to `.never` has other side UX effects? https://github.com/mozilla-lockbox/lockbox-ios/pull/834/files#diff-5860fdbf012da6ac5bba53857bc50c98R93